apps/services/auth-api/src/app/modules/grant-types/grant-types.controller.ts

```diff
@@ -14,16 +15,22 @@ import {
   IdsAuthGuard,
 } from '@island.is/auth-api-lib'
 
-@UseGuards(IdsAuthGuard, ScopesGuard)
+// TODO: Add guards after getting communications to work properly with IDS4
+// @UseGuards(IdsAuthGuard, ScopesGuard)
 @ApiTags('grants')
 @Controller('grants')
 export class GrantTypeController {
```

Comment:
Actually, grant types API is not needed for identityserver, so this should probably be moved to auth-admin-api service.